### ISSUE TITLE
can't add shadow to ToggleSwitchButton

### ISSUE THREAD

**Issue by ksobon** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#958]
My control is defined like so: 

```
<Controls:ToggleSwitchButton Name="AddInputButton"
                                     Grid.Row="1"
                                     Width="48" 
                                     Height="48" 
                                     HorizontalAlignment="Right" 
                                     VerticalAlignment="Bottom"
                                     Margin="0,0,20,20"
                                     Style="{DynamicResource MaterialDesignActionToggleButton}"
                                     IsChecked="{Binding AddInputChecked, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
                                     Visibility="{Binding IsChecked, RelativeSource={RelativeSource Self}, Converter={StaticResource BooleanToVisibilityInverseConverter}}"
                                     materialDesign:ShadowAssist.ShadowDepth="Depth2" Grid.RowSpan="2">
            <Viewbox Width="36" Height="36">
                <Canvas Width="24" Height="24">
                    <Path Data="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z" Fill="White" />
                </Canvas>
            </Viewbox>
        </Controls:ToggleSwitchButton>
```

The result is this: 

![image](https://user-images.githubusercontent.com/529781/40553442-a97d3198-6010-11e8-8a42-6747e625882a.png)

**Comment by Keboo** [comment]
This may be a bug. The [MaterialDesignActionToggleButton](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/master/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ToggleButton.xaml#L25) style ignores the shadow. I suspect it probably has something to do with the flip animation when it is clicked, but will need to spend some time testing to find out. 

In your screenshot, it looks like you are using it like an action button. If that is the case you might consider using one of the of the [*ActionButton styles or perhaps the PopupBox](MaterialDesignMultiFloatingActionPopupBox)?

**Comment by Keboo** [comment]
@Streissi are you still wanting to look at doing this?

**Comment by Streissi** [comment]
@Keboo i only ned to do a pullrequest

**Comment by Keboo** [comment]
Fixed by #1038
